src/createVersion.js

```diff
@@ -15,7 +15,7 @@ function createCompatibleVersion({ currentVersions, allVersions, compatibleWith
     return null;
   }
 
-  const maxCompatible = semver.maxSatisfying(allVersions, `^${compatibleWith}`);
+  const maxCompatible = semver.maxSatisfying(allVersions, `>${compatibleWith}`);
```

Comment:
Given the case:
existing tags: 0.1.0, 0.2.0, 1.0.0, 2.0.0
compatibleWith: 0.0.0
What is the expected output? What would this code return?
I think we have another edge case with the following:
existing tags: none
targetVersion: 0.1.0
compatibleWith: 0.1.0
What would the output be?

Reply:
Maybe adding a couple of unit tests would help..?